test_runner/src/test/kotlin/ftl/args/yml/ErrorParserTest.kt

```diff
@@ -78,6 +83,34 @@ Error node: {
         Assert.assertEquals(exceptedMessage, actualMessage)
     }
 
+    @Test
+    fun `Should properly build error message from MarkedYAMLException`() {
```

Comment:
Rest of functions starts from lower letter.

Reply:
fixed